core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -320,22 +385,22 @@ private void fix(Attributes atts, List<PluginWrapper.Dependency> optionalDepende
         }
     }
 
-    private static final List<DetachedPlugin> DETACHED_LIST = Arrays.asList(
-        new DetachedPlugin("maven-plugin","1.296","1.296"),
-        new DetachedPlugin("subversion","1.310","1.0"),
-        new DetachedPlugin("cvs","1.340","0.1"),
-        new DetachedPlugin("ant","1.430.*","1.0"),
-        new DetachedPlugin("javadoc","1.430.*","1.0"),
-        new DetachedPlugin("external-monitor-job","1.467.*","1.0"),
-        new DetachedPlugin("ldap","1.467.*","1.0"),
-        new DetachedPlugin("pam-auth","1.467.*","1.0"),
-        new DetachedPlugin("mailer","1.493.*","1.2"),
-        new DetachedPlugin("matrix-auth","1.535.*","1.0.2"),
-        new DetachedPlugin("windows-slaves","1.547.*","1.0"),
-        new DetachedPlugin("antisamy-markup-formatter","1.553.*","1.0"),
-        new DetachedPlugin("matrix-project","1.561.*","1.0"),
-        new DetachedPlugin("junit","1.577.*","1.0")
-    );
+    private static final List<DetachedPlugin> DETACHED_LIST = Collections.unmodifiableList(Arrays.asList(
+            new DetachedPlugin("maven-plugin", "1.296", "1.296"),
```

Comment:
NIT: IIUC, only the immutability was added above, the rest is reformatting right?